src/components/nav-drawer/nav-drawer.spec.ts

```diff
@@ -165,7 +165,7 @@ describe('Navigation Drawer', () => {
 
     it('should successfully cancel opening event', async () => {
       el.addEventListener('igcOpening', (ev: CustomEvent) => {
-        ev.detail.cancel = true;
+        ev.preventDefault();
```

Comment:
The test has 2 checks at the end:
      expect(eventSpy).calledWith('igcOpening');
      expect(el.open).to.equal(false);
The first one would pass even if you don't prevent the event. I suggest we change it to:
expect(eventSpy).calledOnceWith('igcOpening');
This way the check would fail because emitEvent would be called twice - once with igcOpening & once with igcOpened.
The other option of course is to just remove this spy check.